src/Raven.Server/Documents/PeriodicBackup/PeriodicBackup.cs

```diff
@@ -27,6 +31,35 @@ public class PeriodicBackup
 
         public PeriodicBackupStatus RunningBackupStatus { get; set; }
 
+        public PeriodicBackup(Logger logger)
+        {
+            _disposeOnce = new DisposeOnce<SingleAttempt>(() =>
+            {
+                DisableFutureBackups();
+
+                if (RunningTask?.IsCompleted == false)
+                {
+                    RunningTask.ContinueWith(t =>
```

Comment:
I don't like it very much.
You are disposing, but the backing is _running_ right now, so you just leaving it to continue running?